### ISSUE TITLE
l2tp dont connect after a while (30min) security error

### ISSUE THREAD

**Issue by bobyturk** [SoftEtherVPN/SoftEtherVPN#1770]
Hi, there!
there is 2 softether DE version using the cascade connection for tunneling . openvpn and sstp is working good . but l2tp connection just work few minutes after server starts . then all users got security layers error and cant connect untill the first server restart again . after restart again l2tp connects just few minutes for example 30 minutes then got error

**Comment by domosekai** [comment]
When l2tp is down, please check if udp port 500 is opened for listening. On Linux run `ss-nulp` as admin to find out.

**Comment by bobyturk** [comment]
i checked and the port 500 is active

UNCONN 0      0                             127.0.0.1:500          0.0.0.0:*     users:(("vpnserver",pid=1001,fd=77))
UNCONN 426496 0                          x.x.x.x:500          0.0.0.0:*     users:(("vpnserver",pid=1001,fd=72))

**Comment by domosekai** [comment]
Is `x.x.x.x` the correct interface for connecting?

**Comment by domosekai** [comment]
We have spotted an issue #1772  in UDP listener but I am not sure if it is related because the information you provided is so little.

**Comment by bobyturk** [comment]
> Is `x.x.x.x` the correct interface for connecting?

yes it is the servers ip . sstp and openvpn work correctly without any error

**Comment by AMajor-C** [comment]
+1 in last build ipsec is totally broken

**Comment by bobyturk** [comment]
i checked that . port 500 is open and listening

but i figured out somthing else . when cpu usage got high then l2tp didnt connect . just like image i attached 

cpu usage got high without any reason and anything dosent change even online users or network usage ... just go high

![photo_2023-03-02_21-23-55](https://user-images.githubusercontent.com/87282588/222512031-2d5046e5-1078-4e4f-a481-779472e9a6f2.jpg)

**Comment by AMajor-C** [comment]
![изображение](https://user-images.githubusercontent.com/7958892/224539831-bb1dc14c-7dd7-45ad-9c70-3016ec87b741.png)
that is going on when ipsec is enabled and we are trying to connect from mikrotik over l2tpv3
